interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/waiting-room-patient.twig

```diff
@@ -0,0 +1,32 @@
+<div class="card-header">
+    <div class="row">
+        <div class="col-md-3 telehealth-user-icon">
+            <i class="fa fa-user-md"></i>
+        </div>
+
+        <div class="col-md-9">
+            <h5 class="text-primary">{{ provider.title }} {{ provider.fname }} {{ provider.mname }} {{ provider.lname }}</h5>
+            {% if provider.specialty is not empty %}
+            <h6 class="text-secondary">{{ "Specialty"|xlt}}: {{ provider.specialty|text}}</h6>
+            {% endif %}
+        </div>
+    </div>
+</div>
+<div class="card-body">
+    <div class="row">
+        <div class="col">
+            <h5>{{ "Appointment Agenda"|xlt }}</h5>
+            <p>{{ appointment.notes }}</p>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col">
+            <input disabled class="disabled btn btn-primary btn-comlink-conference-join d-none d-lg-inline-block" type="button" value="{{ "Join Now"|xla }}">
+        </div>
+    </div>
+    <div class="row d-none" class="waiting-room-server-communication">
+        <div class="col">
+            <p class="alert alert-danger"{{ "We are having problems communicating with the server. Please wait or contact customer support"|xl }}></p>
```

Comment:
xlt

Reply:
also, the string is in wrong place above and is not between the `<p>` and `</p>` tags

Reply:
good catch.

Reply:
fixed the p tag and escaping.